measuring/questions/who-uses.md

```diff
@@ -0,0 +1,15 @@
+# Who uses the InnerSource project?
```

Comment:
Is "project" too large?  "Component" may be too large.

Reply:
There is some hierarchy summary - "Reuse pyramid".  @gruetter may know more, here.

Reply:
We can't think of anything better.